eGovPDX/portlandor
# POWR-1557 flatten pl repo into theme, single source webpack build, clean up scr…

…ipts, organize dirs

Testing:

To run PL(Old):
Note: We cannot currently run PL from a lando container as we do not have a container that supports both node and php. 
1. Go to: `/web/themes/custom/cloudy`
2. Run `npm start`
3. Go to `http://127.0.0.1:3000/` in your browser to see PL. You should see a fully built site with side bar menu, elements, components, etc.
3.b For a faster bootup, after you've run `npm start` the first time, run `npm run dev` instead

To run PL(New):
2. Run `lando npm run build`
3. Go to  `https://portlandor.lndo.site/themes/custom/cloudy/dist-pl/index.html`

Testing PL
1. In the sidebar-menu, navigate to components>links
2. These links should be stylistically the same as the links found here: https://beta.portland.gov/bps/design-guidelines

Testing the Site
1. Go to the multidev associated with this PR
2. There should be no stylistic regression. If there is a problem it should be a significant visible difference

Next Steps:
1. Remove Tailwind
2. Separate out tempates and PL-demos
